src/gui_common/ChemicalEquation.cs

```diff
@@ -85,6 +85,12 @@ public bool ShowSpinner
         }
     }
 
+    public bool ShowToggle
+    {
+        get => toggleProcess!.Visible;
+        set => toggleProcess!.Visible = value;
```

Comment:
This needs safeguards added. Check how the other GUI things support properties that may be set before the property is set (in short: toggleProcess needs to be nullable and a method for applying its state is needed that is called here and in _Ready).

Reply:
How about:
```suggestion
        public bool? ShowToggle
{
    get
    {
        if (toggleProcess == null)
            return false;

        return toggleProcess.Visible;
    }
    set
    {
        if (toggleProcess == null || value == null)
            return;

        toggleProcess.Visible = (bool)value;
    }
}
```

Reply:
The get should have an underlying field that can be returned. The set should set the underlying field which is then applied later. Right now the property *will ignore* a value if set too early. That's a super unexpected effect.